sys-apps/s6-rc/s6-rc-0.5.4.3-r1.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit optfeature toolchain-funcs
+
+DESCRIPTION="Service manager for the s6 supervision suite"
+HOMEPAGE="https://www.skarnet.org/software/s6-rc/"
+SRC_URI="https://www.skarnet.org/software/${PN}/${P}.tar.gz"
+
+LICENSE="ISC"
+SLOT="0/$(ver_cut 1-2)"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="system-init"
```

Comment:
Sorry, but I don't understand why is it necessary to add **unused** `IUSE`? I am able to switch the system to have `s6-rc` as a service-manager with current `virtual/service-manager`.